components/layout_2020/flow/mod.rs

```diff
@@ -242,10 +242,9 @@ impl BlockFormattingContext {
 
         IndependentLayout {
             fragments: flow_layout.fragments,
-            content_block_size: (flow_layout.content_block_size +
+            content_block_size: Au::from(flow_layout.content_block_size) +
                 flow_layout.collapsible_margins_in_children.end.solve() +
-                clearance.unwrap_or_else(Au::zero).into())
-            .into(),
+                clearance.unwrap_or_else(Au::zero),
```

Comment:
I think clippy might complain about using or else when `Au` has a [`Default`](https://docs.rs/app_units/latest/app_units/struct.Au.html#impl-Default-for-Au) implementation that sets it to zero. You could do something like:
```suggestion
                clearance.unwrap_or_default(),
```